### PR TITLE
[#22] Makefile for Dunner

### PR DESCRIPTION
**Pull request by championballer**
Works on #22 

Added MakeFile for Dunner with dep and install under all and separate target for test. Please see if any changes are required. :+1:

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,16 @@
+#Go parameters
+GOCMD=go
+GOINSTALL=$(GOCMD) install
+GOTEST=$(GOCMD) test
+DEP=dep 
+
+all : dep install
```
[the remark this answers is not in the document]
Comment (championballer) [reply Makefile]: Alright. 👍

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+dep:
+	@$(DEP) ensure
+
+install:
```
[the remark this answers is not in the document]
Comment (championballer) [reply Makefile]: I will check and let you know and accordingly update this. Thanks. 👍

File: Makefile (hunk 1 above)
[the remark this answers is not in the document]
Comment (championballer) [reply Makefile]: Added phony targets @agentmilindu, please check. :+1:

File: Makefile (hunk 2 above)
[the remark this answers is not in the document]
Comment (championballer) [reply Makefile]: @agentmilindu I couldn't find anything conclusive on this, but this approach makes sense to me,so I updated the commit accordingly. Please have a look, and as we move forward we will make changes as per requirements? :+1:

[the remark this answers is not in the document]
Comment (agentmilindu) [reply Makefile]: Cool

**Pull request merged**